templates/rhpam70-trial-ephemeral.yaml

```diff
@@ -286,7 +282,7 @@ objects:
           - name: MAVEN_REPO_USERNAME
             value: "${MAVEN_REPO_USERNAME}"
           - name: MAVEN_REPO_PASSWORD
-            value: "${MAVEN_REPO_PASSWORD}"
+            value: "${DEFAULT_PASSWORD}"
```

Comment:
I'd leave the generated maven_repo_password

Reply:
I actually agree with @sutaakar that this should be replaced with default_password, since it is a trial template where all passwords use this default password, so we should stay consistent.